v_3_1_1/types.go

```diff
@@ -82,6 +82,17 @@ type HyperkubeDocker struct {
 	CommandExtraArgs []string
 }
 
+type HyperkubePod struct {
+	HostExtraMounts  []HostMount
+	CommandExtraArgs []string
+}
+
+type HostMount struct {
```

Comment:
I would like to have types prefixed with their "parent" types. Something like `HyperkubePodHostMount` in this case. I know the name is ugly. But when you remove "parent" type you know what "child" types should removed so we don't have garbage left.